defaults/main.yml

```diff
@@ -39,7 +55,7 @@ sensu_api_password: secret
 sensu_api_uchiwa_path: ''
 sensu_api_timeout: 5000
 sensu_client_config: client.json.j2
-sensu_rabbitmq_config: rabbitmq.json.j2
+sensu_rabbitmq_config: sensu-rabbitmq.json.j2
```

Comment:
What is this trying to be consistent with? None of the other config files use the `sensu-` prefix.